static/js/chart/draw_choropleth.ts

```diff
@@ -75,12 +98,12 @@ function drawChoropleth(
   // Combine path elements from D3 content.
   const mapContent = map.selectAll("path").data(geoJson.features);
 
-  // Scale and center the map.
-  const projection = d3
-    .geoAlbersUsa()
-    .fitSize([chartWidth - LEGEND_WIDTH, chartHeight], geoJson);
+  const projection = geo.geoAlbersUsaTerritories();
   const geomap = d3.geoPath().projection(projection);
 
+  // Scale and center the map
+  fitSize(chartWidth - LEGEND_WIDTH, chartHeight, geoJson, projection, geomap);
```

Comment:
is this something required for all maps, or only puerto rico? asking since it looks like an expensive operation (e.g. calling bounds on the geojson)

Reply:
this is something we would need for all maps to scale and center them. We used to do this for all maps with a built in d3-geo function

Reply:
ah yes, sorry i missed that earlier